cowprotocol/cowswap
# [#3] refactor(orders-table): rename OrdersWidget to OrdersTableWidget

# Summary

Fixes #2579

Essentially, it's just auto-refactoring.
I renamed `OrdersWidget` to `OrdersTableWidget`.
And renamed all entities inside of `modules/ordersTable` from `*limitOrders*` to `*ordersTable*`

  # To Test (in Limit orders)

1. Orders table displaying (Open/History)
2. Orders table pagination (URL persistence, loading)
3. Order receipt modal (should open and display relevant data)
4. Orders cancellation (single/multiple)
